quinn-proto/src/tests/mod.rs

```diff
@@ -352,9 +352,10 @@ fn reject_self_signed_server_cert() {
     info!("connecting");
     let client_ch = pair.begin_connect(client_config_with_certs(vec![]));
     pair.drive();
-    assert_matches!(pair.client_conn_mut(client_ch).poll(),
+    let res = pair.client_conn_mut(client_ch).poll();
+    assert_matches!(res,
```

Comment:
I pulled the `res` out of `assert_matches!` while debugging. Probably it could be reverted, but I mildly prefer this to make it easier to breakpoint and examine